components/Op/OpShortForm.js

```diff
@@ -43,15 +43,16 @@ class OpShortForm extends Component {
           // op.subtitle = values.subtitle
           // op.tags = values.tags
           op.duration = values.duration
-          op.locations = values.locations
-          delete op.location
+          // op.locations = values.locations
```

Comment:
we still should set the general location based on the address. otherwise the location based recommendations won't work.

Reply:
Yes, but which kind of data I should set? the suburb level? or the city level? or the region level data?

Reply:
City and region